lib/app.js

```diff
@@ -689,28 +699,12 @@ module.exports = {
                 .child('hosting/versions')
                 .child(settings.firebase)
                 .push();
-        var bar = null;
-        var total = 0;
         directoryRef.on('value', function(snapshot) {
           var status = snapshot.child('status').val();
-          if (status === 'deploying') {
```

Comment:
Correct, the new delete endpoint won't ever go through the 'deploying' state, instead straight to 'removed' state, and so we don't need the progress bar